app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str) -> None:
+    line_command = command.split(" ")
+
+    if (len(line_command) == 3
+            and line_command[0] == "cp"
+            and line_command[1] != line_command[2]):
+
+        with (open(line_command[1], "r") as file_in, 
+              open(line_command[2], "w") as file_out):
+            file_out.write(file_in.read())
+    
```

Comment:
Empty lines not fixed
<img width="343" alt="image" src="https://github.com/mate-academy/py-copy-file/assets/65166203/c191308b-930f-4664-a8f3-b3228a99355f">

Reply:
I must delete 11 line?